ember-toucan-core/package.json

```diff
@@ -57,9 +57,9 @@
     "@embroider/addon-dev": "^3.0.0",
     "@glimmer/component": "^1.1.2",
     "@glimmer/tracking": "^1.1.2",
-    "@glint/core": "^0.9.7",
-    "@glint/environment-ember-loose": "^0.9.7",
-    "@glint/template": "^0.9.7",
+    "@glint/core": "^1.0.0-beta.3",
```

Comment:
beta versions?

Reply:
Yeah! Pairing with @NullVoxPopuli we hit a few issues with the current version that this version seemed to solve.  By the time we release, maybe `@glint/core` will be proper 1.0.0?

Reply:
yeah, the beta.3 is _very_ close to full 1.0.0